configure.py

```diff
@@ -549,6 +549,9 @@ def find_ninja():
     'test/boost/wasm_alloc_test',
     'test/boost/wasm_test',
     'test/boost/wrapping_interval_test',
+    'test/ldap/ldap_connection_test',
+    'test/ldap/role_manager_test',
+    'test/ldap/saslauthd_authenticator_test',
```

Comment:
could you please port the corresponding change in auth/CMakeLists.txt, test/CMakeLists.txt and test/ldap/CMakeLists.txt from https://github.com/scylladb/scylla-enterprise/pull/5062 ?

Reply:
I applied the diff from the PR. How can I verify cmake works? When I tried, I got an error:
```
// in scylladb.git root dir:
mkdir cmake2
cd cmake2
cmake ..
make
// fails with this:
[  0%] Updating SCYLLA-PRODUCT-FILE, SCYLLA-VERSION-FILE and SCYLLA-RELEASE-FILE
[  0%] Check release with SCYLLA-VERSION-GEN
[  0%] Built target scylla-version-gen
[  0%] Generating ../gen/rust/wasmtime_bindings.hh, ../gen/rust/wasmtime_bindings.cc
cxxbridge: Failed to write file `/home/piotrsmaron/src/scylla/cmake2/gen/rust/wasmtime_bindings.hh`

Caused by:
    No such file or directory (os error 2)
make[2]: *** [rust/CMakeFiles/wasmtime_bindings.dir/build.make:78: gen/rust/wasmtime_bindings.hh] Error 1
make[1]: *** [CMakeFiles/Makefile2:13454: rust/CMakeFiles/wasmtime_bindings.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```